### PR TITLE
Feature/add recommendations to vscode

### PR DESCRIPTION
**Pull request by BasLijten**
I added the "extensions.json" file to .vscode. The recommendations from the documentation will automatically show up in the vscode extensions overview:

image

Also changed the contibution branch from "contrib" to "develop" after a short discussion with Jeff L'Heureux, as that is the active branch for contributions (according to him)

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (jflheureux) [inline .vscode/extensions.json:1]: I think that file should also be added in the following locations as we often open these projects separately in VSCode:
- Website/.vscode
- Website/src/rendering/.vscode
- demo/experience/cdp/guestDataGenerator/.vscode
- kiosk/.vscode
- tv/.vscode

Then we should add an entry in the `compare-synced-files.ps1` file comparison script to ensure all the copies of the file are in sync.

Thanks!

**Review by jflheureux — CHANGES_REQUESTED** [review]
See other comment

File: .vscode/extensions.json (hunk 1 above)
Comment (BasLijten) [reply .vscode/extensions.json:1]: Made changes as requested

**Comment by jflheureux** [comment]
Replaced by #500 which includes these changes.

**Pull request closed without merging**